### ISSUE TITLE
Don't demand use of specific tools in 'Use plain English'

### ISSUE THREAD

**Issue by ElenaFdR** [publiccodenet/standard#391]
In 'How to test', we say:

```
* Test the documentation for grammar using [Grammarly](https://www.grammarly.com/).
* Test the documentation for readability using [Hemingway text editor](https://hemingwayapp.com/).
```

I suggest we soften this by adding 'or a similar tool' to each.

**Comment by ericherman** [comment]
It would be nice to have choices ("such as Foo or Bar") for any non-FOSS tools we recommend, but that's kind of a stretch goal ... "or similar" is an improvement for certain.

**Comment by Ainali** [comment]
I just discovered [LanguageTool](https://languagetool.org/) which is open source and similar to Grammarly.

**Comment by bvhme** [comment]
@Ainali how are your experiences with that? Good enough to flat out replace Grammarly with it?

**Comment by Ainali** [comment]
> @Ainali how are your experiences with that? Good enough to flat out replace Grammarly with it?

I installed the browser plugin and am so impressed by even the free version that I am about to buy the Pro-version just to support it. I can highly recommend you to try out the browser plugin!

**Comment by Ainali** [comment]
This was fixed by d6bf35775fd1a7a8ba2216ce8853632a276987b9 (in #672).
